src/components/FormRenderer/index.tsx

```diff
@@ -57,13 +60,17 @@ const componentMapper = {
 
 export interface FormRendererProps {
     /** A schema which defines structure of the form. */
-    schema: any;
+    schema: Schema;
     /** An object of fields names as keys and values as their values. */
-    initialValues?: any;
+    initialValues?: object;
     /** A submit callback which receives two arguments: values and formApi. */
-    onSubmit: (values: any) => void;
+    onSubmit: (values: object, form: FormApi<object>) => void;
     /** A cancel callback, which receives values as the first argument. */
     onCancel?: () => void;
+    /** Label for cancel button */
+    cancelLabel?: string;
```

Comment:
Is it because the submitLable and cancelLable is not included in the *Schema* anymore? Not sure how we gonna support backward compability?

Reply:
@jessieweiyi yes, this is because of this. And this is the reason why i replaced `header` with `title`. This is provided `Schema` type.
```js
interface Schema {
  title?: ReactNode;
  description?: ReactNode;
  fields: Field[];
}
```

idk, but if we need keep backward compatibility i can create custom type for `Schema`. i can make this changes if you are ok with this

Reply:
Thank you. Let us redefine the type by using Omit: 
```
type Schema = Omit<BaseSchema, "title"> & {
    header: BaseSchema["title"],
}
```